### PR TITLE
Feature/upgrade jdk11 revisted

### PR DESCRIPTION
**Pull request by jaserud**
upgrade to jdk 11 revisited.

reference issue #425

### CODE REVIEW

File: song-server/src/test/java/bio/overture/song/server/service/AnalysisServiceTest.java [hunk 1]
```diff
@@ -729,9 +730,11 @@ public void testVariantCallAnalysisMissingSamplesException() {
   @Transactional
   public void testAnalysisIdDneException(){
     val nonExistentAnalysisId = analysisGenerator.generateNonExistingAnalysisId();
-    assertSongError(() -> service.checkAnalysisAndStudyRelated(DEFAULT_STUDY_ID, nonExistentAnalysisId),
+    SongErrorAssertions
```
Comment (rtisma) [inline song-server/src/test/java/bio/overture/song/server/service/AnalysisServiceTest.java]: static import

File: song-server/src/test/java/bio/overture/song/server/service/AnalysisServiceTest.java [hunk 2]
```diff
@@ -729,9 +730,11 @@ public void testVariantCallAnalysisMissingSamplesException() {
...
+        .assertSongErrorRunnable(() -> service.checkAnalysisAndStudyRelated(DEFAULT_STUDY_ID, nonExistentAnalysisId),
         ANALYSIS_ID_NOT_FOUND);
-    assertSongError(() -> service.checkAnalysisExists(nonExistentAnalysisId), ANALYSIS_ID_NOT_FOUND);
+    SongErrorAssertions
```
Comment (rtisma) [inline song-server/src/test/java/bio/overture/song/server/service/AnalysisServiceTest.java]: seems like this static method is used in alot of places and not statically imported. go through all the files and statically import it

File: song-client/src/main/bin/sing [hunk 3]
```diff
@@ -47,7 +46,7 @@ current_dir=$PWD
 fn=$0
 sing_home=$current_dir/`dirname $fn`/../
 #echo "SING_HOME = $sing_home"
```
Comment (rtisma) [inline song-client/src/main/bin/sing]: remove comment

**Comment by andricDu** [comment]
Good work!

**Pull request merged**